src/db.ts

```diff
@@ -0,0 +1,20 @@
+import { MongoClient } from 'mongodb';
+import { Db } from 'mongodb';
+
+let db: Db;
+
+export const connectToDb = async (cb: Function) => {

```

Comment:
Better not to use Function as a type. 

```suggestion
type ConnectCallback = (err: Error | null) => void;

export const connectToDb = async (cb: ConnectCallback) => {

```

Reply:
done